www/js/services/barcode_service.js

```diff
@@ -1,26 +1,65 @@
-app.service('BarcodeService', function ($q) {
+(function (app) {
+  'use strict';
 
-  var BarcodeService = function () {
-    self = this;
-    this.user = null;
+  var WebScanner = function () {
+    this.scan = function (success, fail) {
+      //success('89317593q4oaosjo182yo4wi');
+      success(WebScanner.DEFAULT_WEB_SCAN);
+    }
   };
+  WebScanner.DEFAULT_WEB_SCAN = {text: "HTTP://NEW.RC4.ME/AAK.NyCBBlUF1qWNZ2k", format: "QR_CODE", cancelled: false};
```

Comment:
If this is from a test card, please put a comment describing the card (name, type).